core/include/scipp/core/dataset.h

```diff
@@ -1052,10 +1052,10 @@ class SCIPP_CORE_EXPORT DataArray {
     return copy(get().slice(slice1, slice2, slice3));
   }
 
-private:
   DataConstProxy get() const;
   DataProxy get();
```

Comment:
I don't think these should be made public? Why is it needed? There is an implict type conversion which should be sufficient.

Reply:
Yes, this wasn't needed. I did look to see if this was implicitly convertible beforehand but it wasn't obvious that it was.